webapp/app.py

```diff
@@ -147,10 +149,24 @@ def global_template_context():
 
     return {"version": VANILLA_VERSION, "versionMinor": version_minor, "path": flask.request.path}
 
+import markupsafe
+
+
+# requested_component =
+
+def class_reference(component=None):
+    with open("build/classreferences.yaml") as data_yaml:
+        data = yaml.load(data_yaml, Loader=yaml.FullLoader)
+
+    component = component or urllib.parse.urlsplit(flask.request.path).path.split('/')[-1]
```

Comment:
This works on an assumption that component name is the last part of the URL, this can fail when we start having more complicated urls for new website.